internal/pkg/aws/route53/errors.go

```diff
@@ -16,11 +16,11 @@ func (e *ErrDomainHostedZoneNotFound) Error() string {
 	return fmt.Sprintf("hosted zone is not found for domain %s", e.domainName)
 }
 
-// ErrDomainNotFound occurs when the domain is not found in the account.
-type ErrDomainNotFound struct {
+// ErrDomainNotFoundInRoute53 occurs when the domain is not found in the account.
+type ErrDomainNotFoundInRoute53 struct {
```

Comment:
I preferred the name before because it read `route53.ErrDomainNotFound` vs. `route53.ErrDomainNotFoundInRoute53` (see: https://github.com/golang/go/wiki/CodeReviewComments#package-names)